src/app/static/src/js/components/runReportTabs/runningReportsDetails.vue

```diff
@@ -0,0 +1,185 @@
+    </div>
+</template>
+
+<script lang="ts">
+    import Vue from "vue"
+    import {ReportLog} from "../../utils/types";
+    import {api} from "../../utils/api";
+    import ErrorInfo from "../errorInfo.vue";
+
+    interface Methods {
+        getMetadata: () => void
+    }
+
+    interface Data {
+        reportLog: ReportLog
+        error: string,
+        defaultMessage: string
+    }
+
+    interface Computed {
+        paramSize: number
+        instanceSize: number
+        versionUrl: string
+        reportLogSize: boolean
+    }
+
+    interface Props {
+        reportKey: string
+    }
+
+    const initialReportLog = {
+        email: "",
+        date: "",
+        report: "",
+        instances: {},
+        params: {},
+        git_branch: "",
+        git_commit: "",
+        status: "",
+        logs: "",
+        report_version: ""
+    }
+
+    export default Vue.extend<Data, Methods, Computed, Props>({
+        name: "runningReportsDetails",
+        props: {
+            reportKey: {
+                type: String,
+                required: true
+            }
+        },
+        components: {
+            ErrorInfo
+        },
+        data(): Data {
+            return {
+                reportLog: initialReportLog,
+                error: "",
+                defaultMessage: ""
+            }
+        },
+        computed: {
+            paramSize: function () {
+                return Object.keys(this.reportLog.params).length
+            },
+            instanceSize: function () {
+                return Object.keys(this.reportLog.instances).length
+            },
+            versionUrl: function () {
+                return `/report/${this.reportLog.report}/${this.reportLog.report_version}/`
+            },
+            reportLogSize: function () {
+                const reportValues = Object.values(this.reportLog)
+                    .filter(value => (value && typeof (value) !== "object"))
+                return reportValues.length > 0
+            }
+        },
+        methods: {
+            getMetadata: function () {
+                api.get(`/running/${this.reportKey}/logs/`)
+                    .then(({data}) => {
+                        this.reportLog = data.data
+                        this.reportLog.params = JSON.parse(this.reportLog.params)
+                        this.reportLog.instances = JSON.parse(this.reportLog.instances)
+                        this.error = "";
+                        this.defaultMessage = "";
+                    })
+                    .catch((error) => {
+                        this.error = error;
+                        this.defaultMessage = "An error occurred when fetching metadata";
+                    });
+            }
+        },
+        mounted() {
+            if (this.reportKey) {
+                this.getMetadata()
+            }
+        },
+        watch: {
+            reportKey() {
+                if(this.reportLog) {
```

Comment:
Why have this conditional here? This says you won't fetch a log for a key if you don't already have one. But you maybe won't have one to start with, as relying on parent component to set.